ext/standard/user_filters.c

```diff
@@ -469,12 +475,12 @@ PHP_FUNCTION(stream_bucket_new)
 	bucket = php_stream_bucket_new(stream, pbuffer, buffer_len, 1, php_stream_is_persistent(stream));
 
 	ZVAL_RES(&zbucket, zend_register_resource(bucket, le_bucket));
-	object_init(return_value);
-	add_property_zval(return_value, "bucket", &zbucket);
+	object_init_ex(return_value, stream_bucket_class_entry);
+	zend_update_property(Z_OBJCE_P(return_value), Z_OBJ_P(return_value), ZEND_STRL("bucket"), &zbucket);
 	/* add_property_zval increments the refcount which is unwanted here */
 	zval_ptr_dtor(&zbucket);
-	add_property_stringl(return_value, "data", bucket->buf, bucket->buflen);
-	add_property_long(return_value, "datalen", bucket->buflen);
+	zend_update_property_stringl(Z_OBJCE_P(return_value), Z_OBJ_P(return_value), ZEND_STRL("data"), bucket->buf, bucket->buflen);
+	zend_update_property_long(Z_OBJCE_P(return_value), Z_OBJ_P(return_value), ZEND_STRL("datalen"), bucket->buflen);
```

Comment:
I thought the RFC says you want to use `dataLength`, but it's still `datalen` here?

Reply:
Ah yes, thanks for the reminder, I haven't implemented it yet

Reply:
Done